<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Access Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="loadDeviceListAction"/>
    <addaction name="saveDeviceListAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>&amp;Конфигурирование</string>
    </property>
    <widget class="QMenu" name="menuDslam">
     <property name="title">
      <string>Дслам</string>
     </property>
     <addaction name="editDslamBoardListAction"/>
     <addaction name="updateDslamBoardsInfoAction"/>
    </widget>
    <widget class="QMenu" name="menuSwitch">
     <property name="title">
      <string>Коммутатор</string>
     </property>
     <addaction name="showVlanSwitchAction"/>
     <addaction name="updateVlanSwitchAction"/>
    </widget>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Пакетное обновление данных</string>
     </property>
     <addaction name="updateAllDslamBoardsInfoAction"/>
     <addaction name="updateVlanAllSwitchAction"/>
     <addaction name="updateInfoAllDevicesAction"/>
    </widget>
    <addaction name="addDeviceAction"/>
    <addaction name="editDeviceAction"/>
    <addaction name="removeDeviceAction"/>
    <addaction name="separator"/>
    <addaction name="menuDslam"/>
    <addaction name="menuSwitch"/>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="settingsAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="barsMenu">
     <property name="title">
      <string>Панель инструментов</string>
     </property>
     <addaction name="mainBarAction"/>
     <addaction name="switchBarAction"/>
     <addaction name="dslamBarAction"/>
    </widget>
    <addaction name="barsMenu"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="viewMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Основная панель</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="selectDeviceAction"/>
   <addaction name="separator"/>
   <addaction name="addDeviceAction"/>
   <addaction name="editDeviceAction"/>
   <addaction name="removeDeviceAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="switchToolBar">
   <property name="windowTitle">
    <string>Операции с коммутатором</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="saveConfigSwitchAction"/>
  </widget>
  <widget class="QToolBar" name="dslamToolBar">
   <property name="windowTitle">
    <string>Операции с дсламом</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="upPortAction"/>
   <addaction name="downPortAction"/>
  </widget>
  <action name="aboutAction">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/configure.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="loadDeviceListAction">
   <property name="text">
    <string>Загрузить список устройств</string>
   </property>
  </action>
  <action name="saveDeviceListAction">
   <property name="text">
    <string>Сохранить список устройств</string>
   </property>
  </action>
  <action name="addDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="removeDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/remove.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="editDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="selectDeviceAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Зайти на устройство</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="updateVlanSwitchAction">
   <property name="text">
    <string>Обновить данные о вланах</string>
   </property>
   <property name="toolTip">
    <string>Обновить данные о вланах</string>
   </property>
  </action>
  <action name="showVlanSwitchAction">
   <property name="text">
    <string>Вланы</string>
   </property>
   <property name="toolTip">
    <string>Информация о вланах коммутатора</string>
   </property>
  </action>
  <action name="updateDslamBoardsInfoAction">
   <property name="text">
    <string>Обновить информацию о досках</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию о досках</string>
   </property>
  </action>
  <action name="updateAllDslamBoardsInfoAction">
   <property name="text">
    <string>Обновить информацию о досках на дсламах...</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию о досках на всех dslam'ах...</string>
   </property>
  </action>
  <action name="editDslamBoardListAction">
   <property name="text">
    <string>Редактировать доски</string>
   </property>
   <property name="toolTip">
    <string>Редактировать доски</string>
   </property>
  </action>
  <action name="updateVlanAllSwitchAction">
   <property name="text">
    <string>Обновить информацию о вланах на коммутаторах...</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию о вланах на всех коммутаторах...</string>
   </property>
  </action>
  <action name="saveConfigSwitchAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Сохранить конфигурацию</string>
   </property>
   <property name="toolTip">
    <string>Сохранить конфигурацию</string>
   </property>
  </action>
  <action name="downPortAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/down.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Деактивировать порт</string>
   </property>
   <property name="toolTip">
    <string>Деактивировать порт</string>
   </property>
  </action>
  <action name="mainBarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Основная панель</string>
   </property>
  </action>
  <action name="switchBarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Операции с коммутатором</string>
   </property>
  </action>
  <action name="dslamBarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Операции с дсламом</string>
   </property>
  </action>
  <action name="upPortAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/up.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Активировать порт</string>
   </property>
   <property name="toolTip">
    <string>Активировать порт</string>
   </property>
  </action>
  <action name="updateInfoAllDevicesAction">
   <property name="text">
    <string>Обновить информацию на всех устройствах...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
